src/test/csf/Peer.h

```diff
@@ -298,6 +298,15 @@ struct Peer
         trustGraph.trust(this, this);
     }
 
+    /**
+     * Stubs
+     */
+    std::size_t
+    allowedDisagree()
+    {
+        return 0;
+    }
```

Comment:
I think its important to have the consensus simulation framework testing this change in the same way the production code operates.